applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -375,6 +389,8 @@ static void audio_datapath_presentation_compensation(uint32_t recv_frame_ts_us,
 			break;
 		}
 
+		ctrl_blk.pres_comp.ctr = 0;
+
 		pres_adj_us = ctrl_blk.pres_comp.sum_err_dly_us / PRES_COMP_NUM_DATA_PTS;
 		if ((pres_adj_us >= (BLK_PERIOD_US / 2)) || (pres_adj_us <= -(BLK_PERIOD_US / 2))) {
```

Comment:
What are the `2`'s here and at line 194?
Could these be a `#define`?
There are a lot of these `2`'s throughout the code, can they all be one or more `#defines` for ease of maintenance, or have a single define of `(BLK_PERIOD_US / 2)`?

Reply:
This is a good point, but we need to keep this PR as small as possible, so need to do this change in another PR